src/Github.js

```diff
@@ -29,25 +29,35 @@ class Github {
         const milestone = milestones[0];
         const epicMilestone = await this.getMilestone(token, milestone);
         let issues = [];
+        let epic = [];
         for (const userStory of epicMilestone) {
             let label = `story:${userStory.number}`;
-            let storyIssues = await this.getFullIssues(token, [label], repos);
-            for (var storyIssue of storyIssues) {
-                storyIssue.story = userStory;
+            epic.push({
+                story: userStory,
+                relatedIssues: this.getFullIssues(token, [label], repos)
+            });
+        }
+        Promise.all(epic.map(userStory => userStory.issues));
```

Comment:
Hmm, I don't think `Promise.all` is actually doing anything here... Should it be removed?